docker-compose.identity.yml

```diff
@@ -46,7 +36,7 @@ services:
       userVerifierPrivateKey: "ebba299e6163ff3208de4e82ce7db09cf7e434847b5bdab723af96ae7c763a0e"
       userVerifierPublicKey: "0xbbbb93A6B3A1D6fDd27909729b95CCB0cc9002C0'"
 
-      dbUrl: "postgres://postgres:postgres@identity-service-db:5432/postgres"
+      dbUrl: "postgres://postgres:postgres@db:5432/identity_service"
```

Comment:
@raymondjacobson made it use the same db, and that's why we don't need startup script change